### ISSUE TITLE
Cannot get past column headers with NVDA

### ISSUE THREAD

**Issue by aardrian** [ceolter/ag-grid#3342]
**I'm submitting a ...**  (check one with "x")
```
[X] bug report => see 'Providing a Reproducible Scenario'
[] feature request => do not use Github for feature requests, see 'Customers of ag-Grid'
[] support request => see 'Requesting Community Support'
```

**Customers of ag-Grid**

I am not a customer, but I am evaluating ag-Grid for a client, a potential ag-Grid customer.

**Requesting Community Support**

I understand this will be closed if inactive for one month. However, as a non-customer I have no other way to report bugs.

**Providing a Reproducible Scenario**

1. Go to the [ag-Grid page promoting its accessibility](https://www.ag-grid.com/javascript-grid-accessibility/).

2. Scroll to the bottom and find the only demo on the page, [opening the Plunker in its own window](https://www.ag-grid.com/example-runner/vanilla.php?section=javascript-grid-accessibility&example=accessibility&enterprise=1&generated=1&grid=%7B%22theme%22%3A%22ag-theme-balham%22%2C%22noStyle%22%3A0%2C%22height%22%3A%22100%25%22%2C%22width%22%3A%22100%25%22%2C%22enterprise%22%3A1%7D) (I provide that link for ease of accessing it, but still do step 1 so you have the latest ag-Grid-sanctioned accessible demo).

3. Using NVDA, navigate to the grid (typically by pressing `T`, though it does not always work for this demo).

4. Using `Ctrl` + `Alt` + left or right arrow, navigate the column headings to ensure you are in the grid.

5. From any column header, press `Ctrl` + `Alt` + down arrow to move into the next row (the first data row).

**Current behavior**
NVDA announces "Edge of table" instead of moving to the next row.

**Expected behavior**
I should move to the cell immediately below the column header, NVDA should announce it.

**Please tell us about your environment:**

* **ag-Grid version:** Whatever is the version in your live demo.

* **Browser:**
  * Firefox 69, NVDA 2019.2

* **Language:** Whatever is the language in your live demo.

**Comment by davis4acca** [comment]
Hi,

Your ticket has been flagged as in_zendesk, it means that we recognize this as an issue worth investigating so we have moved it to our official support channel for customers (zendesk)

We will carry on there with the investigation and we will update you as soon as possible.

Thanks

**Comment by kiril-matev** [comment]
Hi @aardrian,

We've significantly improved accessibility since you reported this behavior.

You can verify that the use case you reported is now fully supported - using the NVDA keyboard shortcuts you can move from a column header into the grid rows in AG Grid..

Thank you for reaching out - we appreciate your time!

**Comment by aardrian** [comment]
@kiril-matev 
I confirm I can get into the cell below the header. In that regard, the very specific issue I raised is now addressed.

But after I move down into the table, the cell in which I land does not respond to NVDA table navigation. If I go left or right (`Ctrl` + `Alt + `←` or `Ctrl` + `Alt + `→`) then I hear "Edge of table". If I go down (`Ctrl` + `Alt + `↓`) then I am moved _up_ to the column header.

If I just left arrow or right arrow (`←` or `→`) from a data cell then I move one cell to the left or right. When I press it again, then NVDA does the character navigation I expect.

If I `Tab` into the table, then I cannot move up or down a data row with NVDA table commands after dropping into a cell. I can only move left and right. I can move around with arrows or `Tab`, but then I have lost my NVDA navigation completely. 

I see this uses a `grid` role. It should act like a grid (no tab-stops at all since a grid is a single tab-stop on a page).

So I don't believe the larger issue is addressed -- being able to move more than one cell deep in the table/grid.

Firefox 87, NVDA 2021.3.1
